packages/components/src/FormField/FormFieldAffix.tsx

```diff
@@ -19,6 +19,7 @@ export function AffixLabel({
   const affixLabelClass = classnames(styles.affixLabel, {
     [styles.suffix]: variation === "suffix",
   });
+  if (!label) return null;
```

Comment:
This might actually break things. confirm this

Reply:
Thanks for catching these!